lua/lspconfig/manager.lua

```diff
@@ -1,6 +1,5 @@
 local api = vim.api
 local lsp = vim.lsp
-local uv = vim.uv or vim.loop
```

Comment:
why revert these?

Reply:
Eh, why not? I thought it'd be simpler to just use vim.loop until we don't need it anymore, this compat hack isn't really needed. Can unrevert if you prefer

Reply:
Also I get nervous when I see "uv" because I know we can't use "vim.uv", but then I see that "uv" isn't "vim.uv", we annoyed me.